src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Yes, https://github.com/dotnet/wcf/commit/22998db02e379092318754d728b3b17445d7aa1c added it during https://github.com/dotnet/wcf/pull/646. If you change https://github.com/dotnet/wcf/blob/master/dir.targets#L29 to some property, you could pass in desired prerelease identifiers perhaps with `*` as the default.